server/src/server.ts

```diff
@@ -501,6 +502,20 @@ function documentSymbol(msg: p.RequestMessage) {
   return response;
 }
 
+function workspaceSymbol(msg: p.RequestMessage) {
+  // https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#workspace_symbol
+  let result: p.WorkspaceSymbol[] | null = utils.runAnalysis(
```

Comment:
Why is this not called like all the other comments? Why does it need its own `runAnalysis` function?
Is there some special difference?

Reply:
The `workspace/symbol` request parameters do not have any `textDocument.uri`. The function to perform the analysis requires a `filePath` argument.